macroid-core/src/main/scala/macroid/Searching.scala

```diff
@@ -7,16 +7,17 @@ import android.app.Activity
 import macroid.util.SafeCast
 import macroid.support.{ Fragment, FragmentApi }
 
-/** A class to generate unique ids
-  * The recommended usage is to create a singleton for the entire app:
-  * {{{
-  *   object Id extends IdGen(1000)
-  *   ...
-  *   w[Button] <~ id(Id.button)
-  * }}}
-  *
-  * @param start  The starting id.
-  */
+/**
+ * A class to generate unique ids

```

Comment:
Yea, not sure why that's the default. Maybe sbt-scalariform is out of date. https://github.com/scala-ide/scalariform/blob/b0b09a63bb11b5539a6e22563af0e8c2e23d5e3d/scalariform/src/main/scala/scalariform/formatter/preferences/PreferenceDescriptor.scala#L220